server/models/Group.js

```diff
@@ -250,6 +250,15 @@ export default function(Sequelize, DataTypes) {
     },
 
     instanceMethods: {
+      getUsersForViewer(viewer) {
+        const promises = [this.getUsers()];
```

Comment:
What do you suggest instead?

Not sure why you it's going to miss users/roles. The goal of this instance method on the Group model is to get all the users related to the current group. If a user appears multiple times because it has multiple roles it's fine.